tests/kfto/core/environment.go

```diff
@@ -42,6 +42,10 @@ const (
 	sleepImageEnvVar = "SLEEP_IMAGE"
 	// The environment variable specifying s3 bucket folder path used to store model
 	storageBucketModelPath = "AWS_STORAGE_BUCKET_MODEL_PATH"
+	// The environment variable for the CUDA Training image
```

Comment:
just a very small nit
```suggestion
	// The environment variable for the CUDA training image
```

Reply:
Updated, thank you :)